javascript/javascript2/week2/Doublingnumber.js

```diff
@@ -0,0 +1,6 @@
+let numbers = [1, 2, 3, 4];
+
+const result = numbers
+  .filter((number) => number % 2 !== 0)
+  .map((number) => number * 2);
+console.log(result);
```

Comment:
Good job. Sounds good. 
I have a suggestion, if there is only one input parameter then no need to put it inside the round bracket.

Reply:
i tried to do this but when i save the changes it is back again

Reply:
No problem. It works with brackets as well, Just keep it in mind that for more than one input you need to put it in brackets. But if its one input, and even if you ignore bracket its fine.